src/heretic/p_saveg.c

```diff
@@ -129,9 +129,9 @@ void SV_WritePtr(void *ptr)
 //
 //==========================================================================
 
-void SV_Read(void *buffer, int size)
+int SV_Read(void *buffer, int size)
```

Comment:
Nothing checks this return value, so this is misleading. Instead of fixing the warning you're just finding a different way of ignoring the error. A simple solution would be to check the return value from fread() and call I_Error() if it fails.

Reply:
I was thinking of giving the higher-level code a chance to react instead of just crashing out instantly. But if you want it to terminate, I can do that.